packages/commons/src/conditionals/conditionals.test.ts

```diff
@@ -207,50 +207,36 @@ describe('object combinator', () => {
           dob: field('dob').isBefore().now()
         }),
         getFieldParams({
-          'child.details': {
-            dob: new Date('2125-01-01').toISOString().split('T')[0]
-          }
+          'child.details': new Date('2125-01-01').toISOString().split('T')[0]
```

Comment:
The test is validating the support for `$now` variable even in nested levels, so it should be passing with the previous value. Although as `.object` has been deprecated, what we could do it use `.get('dob')` instead

Reply:
used `.get('data')`